packages/driver/test/cypress/integration/commands/request_spec.coffee

```diff
@@ -48,6 +49,17 @@ describe "src/cy/commands/request", ->
             headers: {
               "x-token": "abc123"
             }
+            timeout: 30000
+          })
+
+      it "accepts object with url + timeout", ->
+        cy.request({url: "http://localhost:8000/foo", timeout: 60000}).then ->
```

Comment:
let's make the value obtuse... other than `60000` so it's more clear that we've altered it in the expected way in our test... maybe something like `12345`